main/bind/APKBUILD

```diff
@@ -240,9 +275,17 @@ libs() {
 	depends="$depends_libs"
 	default_libs
 }
+_append_deps() {
+	local s split
+	for s in $subpackages; do
+		split="$(subpkg_set "$s"; printf -- '%s' "$subpkgsplit")"
+		eval "makedepends=\"\$makedepends \${depends_${split}}\""
+	done
+}
+case "$(type 2>/dev/null subpkg_set)" in (subpkg_set\ is\ a\ *function*) _append_deps ;; esac
```

Comment:
I dont like use `$()` in global scope because it forks. It is not problem of a single APKBUILD but things would go horribly slow if every APKBUILD forked in global scope.

Reply:
This is just a work around for `abuild` not having this sort of thing built-in yet.

Everything is a built-in but I haven't come up with a way to avoid the fork entirely here.

Reply:
I've switched this to a missing function (when sourcing APKBUILD outside abuild) that should let us avoid the sub-shell here.